app/queries/commission_history_query.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class CommissionHistoryQuery
+  QUERY_FIELDS = 'commission_histories.created_at,
```

Comment:
I think you could specify table for each field. Use array, #map with block to add table name and join.